src/PIL/Jpeg2KImagePlugin.py

```diff
@@ -328,6 +328,8 @@ def _save(im, fp, filename):
     mct = info.get("mct", 0)
     signed = info.get("signed", False)
     fd = -1
+    comment = info.get("comment", None)
```

Comment:
```suggestion
    comment = info.get("comment")
```
Just because `None` is the default.